components/authentication/azure/src/main/java/com/microsoft/kiota/authentication/AzureIdentityAccessTokenProvider.java

```diff
@@ -147,7 +147,8 @@ public AzureIdentityAccessTokenProvider(
         }
     }
 
-    @Nonnull public AllowedHostsValidator getAllowedHostsValidator() {
+    @Nonnull 
```

Comment:
you'll probably need to run `./gradlew spotlessApply` to fix the linting check here

Reply:
Thank you for your guidance in resolving this issue. I confirm that I will run `./gradlew spotlessApply` to fix the linting issues in the affected file. Thank you once again for your support.